OpenRA.Mods.Common/Traits/Render/WithVoxelBody.cs

```diff
@@ -23,6 +23,9 @@ public class WithVoxelBodyInfo : UpgradableTraitInfo, IRenderActorPreviewVoxelsI
 	{
 		public readonly string Sequence = "idle";
 
+		[Desc("Defines if the Voxel should have a shadow.")]
+		public readonly bool ShowShadow = true;
```

Comment:
I'm not yet asking you to change this, but i'd like to point out that we are inconsistent in how we name these flags (some traits use Show*, others use Display*).  It would be good if we could decide on a convention, use that here, and then fix the naming of the other cases in a followup PR.